src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -61,6 +62,9 @@
   @JsonProperty
   private long maxBufferSize = DEFAULT_MAX_BUFFER_SIZE;
 
+  @JsonProperty
+  private boolean compress = DEFAULT_COMPRESS;
```

Comment:
for now we only going to support gzip, but it is so common to introduce different compression codecs, do you think it would be handy to have string name for compression algorithm in config instead of just a boolean flag?